coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetViewHelperServiceImpl.java

```diff
@@ -205,4 +213,15 @@ public void setKcBusinessRulesEngine(KcBusinessRulesEngine kcBusinessRulesEngine
 		this.kcBusinessRulesEngine = kcBusinessRulesEngine;
 	}
     
+	public boolean isBudgetMarkedForSubmission(Budget finalBudget, Budget currentBudget) {
+		return getProposalBudgetService().isBudgetMarkedForSubmission(finalBudget, currentBudget);
```

Comment:
More than I was expecting to be passed in but looks good and SO happy to be rid of ViewLifecycle call.